config/routes.rb

```diff
@@ -2,9 +2,10 @@
   devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :users, only: [:index, :show] do
-  resources :foods
-  resources :recipes
+    resources :foods
   end
+  resources :recipes, only: [:index, :show, :new, :create, :destroy]
+  resources :public_recipes, only: [:index]
```

Comment:
## Concern.
Doesn't leave recipe outside the user route, allow people to see not only public recipe but also recipe for a single user without needing to sign in? Also allow user to destroy and create a recipe without being a user?

Reply:
Well I think is better if we use cancanca for manage that